locale.c

```diff
@@ -6031,45 +6104,109 @@ S_strftime_tm(pTHX_ const char *fmt, const struct tm *mytm)
     restore_toggled_locale_c(LC_CTYPE, orig_CTYPE_LOCALE);
 
 #  endif
+
     return buf;
 
 #endif
 
 }
 
-char *
-Perl_my_strftime8_temp(pTHX_ const char *fmt, int sec, int min, int hour, int mday,
-                         int mon, int year, int wday, int yday, int isdst,
-                         utf8ness_t * utf8ness)
-{   /* Documented above */
-    char * retval = my_strftime(fmt, sec, min, hour, mday, mon, year, wday,
-                                yday, isdst);
+STATIC char *
+S_strftime8(pTHX_ const char * fmt,
+                  struct tm * mytm,
```

Comment:
I don't believe this modifies whatever mytm points at, so this could be `const struct tm *` (which matches the POSIX strftime.

Similarly for `sv_strftime_tm()`